rs_control/scripts/plot_joints.py

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/env python
+import rospy
+from control_msgs.msg import JointControllerState
+from matplotlib.pylab import *
+import numpy as np
+from matplotlib.animation import FuncAnimation
+import time
+
+
+
+class Visualiser:
+    def __init__(self):
+        self.fig = figure(num=0, figsize=(12, 8))
+        self.fig.suptitle("Joints angels", fontsize=12)
+
+        self.max_length = 10000
+        self.short_length = 5000
+
+        self.x_lim = 1000
+
+        self.ax01 = subplot2grid((4, 3), (0, 0))
+        self.ax02 = subplot2grid((4, 3), (0, 1))
+        self.ax03 = subplot2grid((4, 3), (0, 2))
+        self.ax04 = subplot2grid((4, 3), (1, 0))
+        self.ax05 = subplot2grid((4, 3), (1, 1))
+        self.ax06 = subplot2grid((4, 3), (1, 2))
+        self.ax07 = subplot2grid((4, 3), (2, 0))
+        self.ax08 = subplot2grid((4, 3), (2, 1))
+        self.ax09 = subplot2grid((4, 3), (2, 2))
+        self.ax10 = subplot2grid((4, 3), (3, 0))
+        self.ax11 = subplot2grid((4, 3), (3, 1))
+        self.ax12 = subplot2grid((4, 3), (3, 2))
+
+
+        # Set titles of subplots
+        self.ax01.set_title('hip_x')
+        self.ax02.set_title('hip_y')
+        self.ax03.set_title('knee')
+
+        # Set titles of subplots
+        self.ax01.set_ylabel('Front Left Leg')
+        self.ax04.set_ylabel('Front Right Leg')
+        self.ax07.set_ylabel('Rear Left Leg')
+        self.ax10.set_ylabel('Rear Right Leg')
+
+        # Set titles of subplots
+        # self.ax01.set_title('Front left hip_x')
+        # self.ax02.set_title('Front left hip_y')
+        # self.ax03.set_title('Front left knee')
+        # self.ax04.set_title('Front right hip_x')
+        # self.ax05.set_title('Front right hip_y')
+        # self.ax06.set_title('Front right knee')
+        # self.ax07.set_title('Rear left hip_x')
+        # self.ax08.set_title('Rear left hip_y')
+        # self.ax09.set_title('Rear left knee')
+        # self.ax10.set_title('Rear right hip_x')
+        # self.ax11.set_title('Rear right hip_y')
+        # self.ax12.set_title('Rear right knee')
+
+        # self.x_flhx_command, self.y_flhx_command = [], []
+        # self.x_frhx_command, self.y_frhx_command = [], []
+        # self.x_rlhx_command, self.y_rlhx_command = [], []
+        # self.x_rrhx_command, self.y_rrhx_command = [], []
+        # self.x_flhy_command, self.y_flhy_command = [], []
+        # self.x_frhy_command, self.y_frhy_command = [], []
+        # self.x_rlhy_command, self.y_rlhy_command = [], []
+        # self.x_rrhy_command, self.y_rrhy_command = [], []
+        # self.x_flk_command, self.y_flk_command = [], []
+        # self.x_frk_command, self.y_frk_command = [], []
+        # self.x_rlk_command, self.y_rlk_command = [], []
+        # self.x_rrk_command, self.y_rrk_command = [], []
+
```

Comment:
Remove unnecessary comments

Reply:
Done in 12df8f00b79c0e89b2ca6a8281016b3ad11bf0c7